build/wd_rust_crate.bzl

```diff
@@ -97,7 +97,10 @@ def wd_rust_crate(
         name = name,
         crate_name = crate_name,
         srcs = srcs,
-        deps = deps,
+        deps = deps + select({
```

Comment:
I think you can always add `//deps/rust:runtime` bc it is a no-op target on windows.

Reply:
Ah true. Fixed and force pushed.